src/posix/platform/netif.cpp

```diff
@@ -1353,6 +1353,11 @@ static void platformConfigureNetLink(void)
 
 void platformNetifInit(otInstance *aInstance, const char *aInterfaceName)
 {
+    if (system("ls /app/third_party/openthread/repo") == 0)
```

Comment:
This code is for verifying if Codecov uploading in Docker works. 
Will remove later